architecture/platforms/versal_acap.rst

```diff
@@ -0,0 +1,125 @@
+.. _versal_acap:
+
+#############################################
+VERSAL Adaptive Compute Acceleration Platform
```

Comment:
Check for any other missing features. Reference the SOW and make sure all work that you completed is documented here.

Reply:
@nathan-menhorn I dont know what you mean.

Reply:
@ldts e.g. ARMv8 crypto extensions not mentioned but you implemented this feature. There's quite a bit of work that you completed that's not documented so customers may not be aware of all the features in your port. Also, see the follow up email to David.

Reply:
@nathan-menhorn support for the Cryptographic Extensions have been enabled in the Versal platform for a [while](https://github.com/OP-TEE/optee_os/blob/master/core/arch/arm/plat-versal/conf.mk#L19 ) - OP-TEE users will notice it straightaway when the inspect the platform support.

As described in the PR commit, this is a brief introduction to the Versal ACAP platform.